app/baseComponents/SendReceiveButton/SendReceiveButton.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+/* eslint react/no-unused-prop-types: 0 */
```

Comment:
remove this comment

Reply:
For some reason, in a functional component, eslint will always comment on "PropType defined but never used".
This is true for all required props... It's either this comment or one comment per prop.
What are your thoughts?